math/big/mod.ts

```diff
@@ -0,0 +1,864 @@
+/************************************** EDITABLE DEFAULTS *****************************************/
```

Comment:
Please list where you've forked this code from, the license, author, and copyright header

Reply:
Should I add `Deno copyright` header behind the `origin copyright` header?